### DISCUSSION TITLE
Sub-menu position

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#612] [General] [answered]
Currently, the sub-menu shows up on the right of the menu. If the button to trigger the menu is on the right side of the page, there isn't enough room for the sub-menu to show up on the right of the menu. Where should the sub-menu show up instead? For example, more icon in a table: 
![image](https://user-images.githubusercontent.com/6718764/88739357-3731e780-d0ef-11ea-9b9d-5eda2a1e614a.png)

**Comment by kate-porter** [comment]
@SiTaggart does the sub-menu auto-position itself in cases such as the one Elaine called out here?

**Reply by SiTaggart** [reply]
Pretty sure the menu should auto position itself based on the position it is in the window.

**Comment by SiTaggart** [comment, marked as answer]
Quick test in the browser shows that the submenu will indeed position itself according to where the menu trigger is in the window.

![image](https://user-images.githubusercontent.com/368249/88740437-fe474200-d0f1-11ea-9f73-35ac47c6a4d6.png)

**Reply by kate-porter** [reply]
@elainelylee looks like it should position itself for ya!
